app/lib/features/search/widgets/quick_jump.dart

```diff
@@ -94,7 +96,12 @@ class QuickJump extends ConsumerWidget {
                 ),
               ),
               onPressed: () {
-                navigateTo(route: Routes.chat);
+                final roomId = ref.watch(roomIdProvider);
+                if (isDesktop) {
+                  navigateTo(target: '/chat/$roomId');
+                } else {
+                  navigateTo(route: Routes.chat);
+                }
```

Comment:
yeah, no, the increasing number of cases where routing is changing depending on whether we are `isDesktop` or not concerns me. I think this is the wrong approach.